<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mercadoLibre</class>
 <widget class="QWidget" name="mercadoLibre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>590</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MERPIDO BUSQUEDA</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#mercadoLibre

{
background-image: url(:/imagenes/FondoMeli.png);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Merpido</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_BusquedaMeli">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border-radius: 10px;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-weight: bold;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Search">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>40</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 15px;
    padding: 10px 20px;
    background-color: #333333;
    color: #FFFFFF;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Lo mas barato y Rapido de MercadoLibre</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="imagenProducto1">
      <property name="styleSheet">
       <string>QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 8px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="datosProducto1">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imagenProducto2">
      <property name="styleSheet">
       <string>QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 8px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="datosProducto2">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="imagenProducto3">
      <property name="styleSheet">
       <string>QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 8px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="datosProducto3">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>541</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="imagenProducto4">
      <property name="styleSheet">
       <string>QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 8px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="datosProducto4">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 10px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>411</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="imagenProducto5">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 8px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="datosProducto5">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: white;
    font-family: &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 12px;
    font-weight: bold;
background-color: black;
                       border: 2px solid white;
                        padding: 5px;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
